notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #Lets query all the observations of 3c273 from the Hubble Space Telescope
</pre><p>Again, I think this can be in a markdown cell above, and you can make explicit that you are now adding the obs_collection and dataproduct_type keywords.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>